include/seqan3/std/concepts

```diff
@@ -522,7 +522,7 @@ concept bool Semiregular = Copyable<T> && DefaultConstructible<T>;
 //!\endcond
 
 /*!\interface   std::Regular
- * \brief       Subsumes std::Semi_regular bool and std::Equality_comparable.
+ * \brief       Subsumes std::Semiregular and std::EqualityComparable.
```

Comment:
Semi**R**egular?

Reply:
no, indeed it is Semi**r**egular :see_no_evil: